norbit-webapp/src/app/layout.tsx

```diff
@@ -3,12 +3,13 @@ import type { Metadata } from 'next'
 import { Inter } from 'next/font/google'
 import {HeaderComponent} from './components/HeaderComponent/HeaderComponent';
 import {Device} from './DeviceManager';
+import { Providers } from '../../redux/provider';
 
 const inter = Inter({ subsets: ['latin'] })
 
 export const metadata: Metadata = {
-  title: 'Create Next App',
```

Comment:
I'd either leave these as-is for now, or edit them to something relevant for our app rather than blanking them. LGTM otherwise